lib/classes/config-schema-handler/resolve-ajv-validate.js

```diff
@@ -60,10 +63,13 @@ const getValidate = async (schema) => {
       throw err;
     }
     const moduleCode = standaloneCode(ajv, validate);
-    await fs.promises.writeFile(cachePath, moduleCode);
+
+    const tmpCachePath = path.resolve(getTmpDirPath(), filename);
+    await fse.outputFile(tmpCachePath, moduleCode);
```

Comment:
I think it'll be cleaner to follow same approach as in some other places as:

1. Get process-specific tmpdir via `process-utils/tmpdir`
2. Write file with regular `fsp.writeFile` to it
3. Safely move a file

We do it when handling deprecation report: https://github.com/serverless/serverless/blob/906f42fd91e8870a7b2fadec467bfbfbf0c59002/lib/utils/log-deprecation.js#L141-L147 (still here we do not need to ensure target dir, as it's already ensured by `ensureDir`)